scripts/build_server

```diff
@@ -14,6 +14,7 @@ Build the specified version of the Oak Runtime server.
 
   -s    Server type:
           - base: base version of the server (default)
+          - ndebug: base version of the server with debugging compiled out
```

Comment:
Should we just call these "debug" and "release", going forward? And basically mention that only the "release" binaries actually enforce IFC and are meant to be used in production.

Reply:
I'd avoid "debug" and "release" as labels, because they're likely to be associated with things like debug symbols, `-O ` flag values etc. (and in retrospect I should maybe have used a better name than `OAK_DEBUG`).   But yes, we should definitely make it clear that only the "logless" binaries are safe for production use with real private data.

(Moved to use `logless` as the build label for the moment.)